src/svgui/svgui.ts

```diff
@@ -250,6 +251,69 @@ export class Pair extends UIElement {
     }
 }
 
+/*
+ * Image for element.
+ * Backbone properties:
+ *     imageUrl: string
+ */
+export class Image extends UIElement {
```

Comment:
Remove all other unnecessary changes in this file (except for adding `Image` class), please.